### ISSUE TITLE
LIKE '%x' gives the wrong results

### ISSUE THREAD

**Issue by pgulutzan** [tarantool/tarantool#3334]
I have Ubuntu 14.04.
I have Tarantool 2.0, pulled from source today.

LIKE '%x' gives the wrong results. Example:
```
tarantool> CREATE TABLE t (s1 char(2) primary key, s2 char(2));
---
...

tarantool> INSERT INTO t VALUES ('AB', 'AB');
---
...

tarantool> SELECT * FROM t WHERE s1 LIKE '%A';
---
- - ['AB', 'AB']
...

tarantool> SELECT * FROM t WHERE s1 LIKE '%C'
         > ;
---
- - ['AB', 'AB']
...

```
I don't know whether this might be related to
a problem with LIKE '_x', as in issue#3251.

**Comment by SudoBobo** [comment]
I've tried to reproduce the problem in 2.1 and it appears that the problem is no more actual, so I close the ticket.

**Comment by Totktonada** [comment]
Yep, it was closed along with #3251.

**Comment by SudoBobo** [comment]
Well, commit message (of the commit which was supposed to close the ticket) contains lines:
```
Сloses #3251
Сloses #3334
```
As we can see, this did not close #3334. It's written [here](https://help.github.com/articles/closing-issues-using-keywords/#closing-multiple-issues) that something like 'closes #34, closes #23' should be used.

**Comment by Totktonada** [comment]
The first letter in 'Closes' in the commit message is the Russian letter 'С', that is why the commit (and #3251 too) was not closed automatically.
